Hotfix/Remove keyword from basic metadata (duplicate predicate)

ewilloughby opened the pull request:
Hey @boveus this just removes keyword from basic_metadata.  Kathryn and Jeremy decided subject_local would use the same field keyword did, but I'd left it in, so it was creating a duplicate predicate message when the server starts.

boveus commented:
Hey Eric it looks like we were using that field in a lot of our tests.  I don't see it on the metadata spreadsheet, so I think it would be safe to remove all references to the keyword field in the form/presenter etc.

ewilloughby commented:
Hey @boveus We don't explicitly use keyword in our tests, but it looks like it comes from stuff the actors are calling.  
![image](https://user-images.githubusercontent.com/4126936/43521646-c6d44824-9564-11e8-902f-f7a49002ad41.png)

Keyword seems to be a bit more deeply integrated into hyrax than I realized, so I restored it and just changed its predicate to one from MADS we'll never use.  I added a card to the backlog to look more at keyword and its usage in other pieces of hyrax like actors and ultimately filesets ,collections etc.

boveus commented:
This is reassuring.  This means that we would have received some other error if we didn't override the basic metadata correctly. Good catch - I should have paid more attention to the error message.
